scripts/ansible/client-auto-update-playbook.yml

```diff
@@ -0,0 +1,65 @@
+- hosts: zeitgeist
+  gather_facts: no
+  become: true 
+  environment: 
+    RELEASE_VERSION: "{{ lookup('env', 'RELEASE_VERSION') }}"
+
+  tasks:
+    - name: Get Zeitgeist services
+      no_log: true
```

Comment:
Why did you decided to use `no_log: true` in all cases? This would make it harder to debug.

Reply:
I added the `no_log` to prevent any of the server IP addresses from being outputted when the task runs. The Zeitgeist repository is public, I believe anyone can look through the actions and see the output from each workflow ran.